### PR TITLE
test(subscription): increase code coverage

### PR DESCRIPTION
**Pull request by mannyrivera2010**
AMLNG-840

**Description**     
Increase Unit Test Coverage

**Acceptance Criteria**   
Unit Test Coverage has been increase
 
**How to test code**    
`make ptest`

**Please Review**     
@aml-development/ozp-backend-team    

**Checklist**
- [x] Read CONTRIBUTING.md
- [x] Write code to complete task 
- [x] Python Code is PEP8 Compliant
- [ ] Add unit tests for new code
- [ ] All tests must pass before merging the pull request
- [ ] At least 2 people reviewed code

### CODE REVIEW

File: ozpcenter/api/subscription/model_access.py [hunk 1]
```diff
@@ -128,10 +128,10 @@ def update_subscription(request_username,
...
     # If subscription_ids >= 1 if means that there is already that subscription that exist
     if len(subscription_ids) >= 1:
-        if subscription_instance.id in subscription_ids:
-            return subscription_instance
-        else:
-            raise Exception('Can not update a subscription that already exist')
+        # if subscription_instance.id in subscription_ids:
+        return subscription_instance
+        # else:
+        #     raise Exception('Can not update a subscription that already exist')  # TODO: Is this possible
```
Comment (mleeBoeing) [inline ozpcenter/api/subscription/model_access.py]: Not possible

**Comment by mleeBoeing** [comment]
:+1:
Tests pass and coverage at 87% now

File: ozpcenter/api/subscription/model_access.py (hunk 1 above)
Comment (mannyrivera2010) [reply ozpcenter/api/subscription/model_access.py]: I removed the comments

**Comment by irizvifearless** [comment]
same results as @mleeBoeing :+1:

**Pull request merged**